Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -62,6 +62,12 @@ extension BrowserViewController: BrowserToolbarDelegate {
         presentViewController(controller, animated: true, completion: nil)
     }
 
+    func didClickReaderMode() {
+        if let tab = tabManager.selectedTab {
+            tab.toggleReaderMode()
```

Comment:
Currently, Browser just holds state and the controller logic, acting on the webView, is all done by BVC. If possible, it'd be nice to keep this toggling logic separate from Browser, and have it in BVC or some ReaderMode class is tied to the webView.